.github/CHANGELOG.md

```diff
@@ -62,11 +86,13 @@ Nariman Saadatmand, [Ilan Tzitrin](https://github.com/ilan-tz)
 
 See full commit details https://github.com/XanaduAI/flamingpy/compare/v0.4.6a1...v0.4.9a1
 
+
 ## Release 0.4.6a1
 
 ### New features since the last private release
 
-* The first Cython function for Monte Carlo sampling, mostly to provide cythonization samples and testbeds, has been added. See [`cpp_mc_loop.pyx`](flamingpy/cpp/cpp_mc_loop.pyx) and [`simulations.py`](flamingpy/benchmarks/simulations.py) for detailes. 
+* This is the initial public release started from the private template and our sister project [FT-Stack](https://github.com/XanaduAI/ft-stack).
+* The first Cython function for Monte Carlo sampling, mostly to provide cythonization samples and testbeds, has been added. See [`cpp_mc_loop.pyx`](flamingpy/cpp/cpp_mc_loop.pyx) and [`simulations.py`](flamingpy/benchmarks/simulations.py) for details. (backward incompatible) 
```

Comment:
```suggestion
* The first Cython function for Monte Carlo sampling, mostly to provide cythonization samples and testbeds, has been added. See [`cpp_mc_loop.pyx`](flamingpy/cpp/cpp_mc_loop.pyx) and [`simulations.py`](flamingpy/benchmarks/simulations.py) for details (backward incompatible). 
```

Reply:
Sorry, @ilan-tz, I may keep the old format. This is because your suggestion, while grammatically more correct, is not consistent with other parts. We can discuss changing the formatting everywhere for the next release.